docs/ruleset_functions/get_wwr.md

```diff
@@ -1,4 +1,6 @@
 
+## get_building_segment_wwr
+
```

Comment:
I would update the name of this funciton to get_overall)building_segment_wwr.
We may need another function to calculate WWR based on space conditioning category (residential, non-residential, semi-heated) depending on how a pending 90.1 interpretation request is resolved

Reply:
As per the discussion below, this function will be used to determine baseline WWR as per Table G3.1.1-1. Do we need to differentiate new/addition/alteration vs existing when counting the surface and fenestration areas? If so, how do we want to apply space.status_type [new, addition, alteration, existing] to zone.surfaces? A zone might have both new and existing spaces. @mkarpman @supriyagoel

Reply:
As per our discussion, RDS will raise warning for new/addition/alteration or existing. This function will not differentiate the status_type of the surfaces and subsurfaces.